File: FJT.ReportViewer/MySqlDBModel/reportviewerparameter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace FJT.ReportViewer.MySqlDBModel
{
    public class reportviewerparameter
    {
        [Key]
        public Guid parameterGUID { get; set; }

        public int reportId { get; set; }

        [StringLength(255)]
        public string reportName { get; set; }

        public string parameterValues { get; set; }

        public bool isDeleted { get; set; }

        [StringLength(255)]
        public string createdBy { get; set; }

        [StringLength(255)]
        public string updatedBy { get; set; }

        [StringLength(255)]
        public string deletedBy { get; set; }

        public DateTime createdAt { get; set; }

        public DateTime? updatedAt { get; set; }

        public DateTime? deletedAt { get; set; }

        public int createByRoleId { get; set; }

        public int? updateByRoleId { get; set; }

        public int? deleteByRoleId { get; set; }

        [ForeignKey("reportId")]
        public virtual reportmaster reportmaster { get; set; }
    }
}

File: fjt.api/models/ReportViewerParameter.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const ReportViewerParameter = sequelize.define('ReportViewerParameter', {
        parameterGUID: {
            allowNull: false,
            primaryKey: true,
            type: Sequelize.UUID
        },
        reportId: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        reportName: {
            type: Sequelize.STRING,
            allowNull: true
        },
        parameterValues: {
            type: Sequelize.STRING,
            allowNull: false
        },
        isDeleted: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        createdBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        updatedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        deletedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        createByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        updateByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        deleteByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        }
    }, {
        tableName: 'reportviewerparameter',
        paranoid: true
    });
    ReportViewerParameter.associate = (models) => {
        ReportViewerParameter.belongsTo(models.ReportMaster, {
            foreignKey: 'reportId',
            as: 'ReportMaster'
        });
    };
    return ReportViewerParameter;
};
